### PR TITLE
feat: added mapproxy-seed deployment based on mapproxy-util latest (1…

### PR DESCRIPTION
**Pull request by CL-SHLOMIKONCHA**
….16)

### CODE REVIEW

File: helm/v1.16.0.yaml [hunk 1]
```diff
@@ -0,0 +1,131 @@
+global:
```
Comment (asafMasa) [inline helm/v1.16.0.yaml]: remove

File: helm/values.yaml [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+      cpu: 100m
+      memory: 128Mi
+
+initContainer:
+  targetPort: 8080
+  image:
+    repository: mapproxinator
+    tag: "latest"
+  env:
+    logLevel: warn
```
Comment (asafMasa) [inline helm/values.yaml:76]: info

**Review by asafMasa — CHANGES_REQUESTED** [review]
Good job!

File: helm/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: mapproxy-seed
+description: A Helm chart for mapproxy seed service
```
[the remark this answers is not in the document]
Comment (CL-SHLOMIKONCHA) [reply helm/Chart.yaml:3]: yes, changed it.

File: helm/Chart.yaml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: mapproxy-seed
```
[the remark this answers is not in the document]
Comment (CL-SHLOMIKONCHA) [reply helm/Chart.yaml:2]: added

File: helm/v1.16.0.yaml (hunk 1 above)
Comment (CL-SHLOMIKONCHA) [reply helm/v1.16.0.yaml]: removed

Comment (CL-SHLOMIKONCHA) [reply helm/v1.16.0.yaml]: + added 'local.yaml' to .gitignore

File: helm/values.yaml (hunk 2 above)
Comment (CL-SHLOMIKONCHA) [reply helm/values.yaml]: fixed

File: config/patch/redis.py [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+            log.debug('exists_key, key: %s' % key)
+            return self.r.exists(key)
+        except redis.exceptions.ConnectionError as e:
+            log.error('Error during connection %s' % e)
+            return False  
+        except Exception as e:
+            log.error('REDIS:exists_key error  %s' % e)
+            return False
+
+    def store_tile(self, tile, dimensions=None):
```
[the remark this answers is not in the document]
Comment (CL-SHLOMIKONCHA) [reply config/patch/redis.py:68]: * docker-mapproxy - done
* Mapproxy (ours) - done
* Official mapproxy - TBD

**Review by asafMasa — APPROVED** [review]

File: config/patch/redis.py (hunk 5 above)
[the remark this answers is not in the document]
Comment (CL-SHLOMIKONCHA) [reply config/patch/redis.py:68]: created a jira issue about the official mapproxy pr

**Pull request merged**